pkg/controller/operators/catalog/operator.go

```diff
@@ -897,6 +897,11 @@ func (o *Operator) handleCatSrcDeletion(obj interface{}) {
 }
 
 func validateSourceType(logger *logrus.Entry, in *v1alpha1.CatalogSource) (out *v1alpha1.CatalogSource, continueSync bool, _ error) {
+	logger = logger.WithField("step", "validateSourceType")
+	logger.Info("validate source type start")
```

Comment:
I think in practice you'll find method book-ending is extremely chatty - on the order of MiB/s logging rate during high churn, especially when the system is just verifying correctness of current state and not changing. Not sure if you really require it. The logs for error cases and outcomes are very useful and will not emit if "nothing" is going on.

Reply:
yeah - I tend to agree. The verbosity is high for fairly low utility. I'll cull most of the non-error logs. At least in these paths. Wondering if just leaving pre-return log would still be a good idea to get signal that it went into the method - even if only for the methods higher up the stack....?

Reply:
okie - see if that sparks joy, or if you reckon it's still too verbose

Reply:
Try it out and see! It might still be too chatty to be useful, but you can't know without running an end-to-end test.